vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -277,7 +277,6 @@
             </div>
             <div v-else>
               <button
-                type="submit"
```

Comment:
Removing `type="submit"` will not submit form

Reply:
Form is also submit, i checked